common/src/KokkosFFT_traits.hpp

```diff
@@ -261,6 +261,41 @@ struct complex_view_type {
   using type = Kokkos::View<complex_type*, array_layout_type, ExecutionSpace>;
 };
 
+template <typename ExecutionSpace, typename Enable = void>
+struct is_AnyHostSpace : std::false_type {};
+
+template <typename ExecutionSpace>
+struct is_AnyHostSpace<ExecutionSpace,
+                       std::enable_if_t<Kokkos::SpaceAccessibility<
+                           ExecutionSpace, Kokkos::HostSpace>::accessible>>
+    : std::true_type {};
```

Comment:
```suggestion
template <typename ExecutionSpace>
struct is_AnyHostSpace : std::integral_constant<bool, Kokkos::SpaceAccessibility<ExecutionSpace, Kokkos::HostSpace>::accessible> {};
```

I think it should be the same ? Note that `std::true_type` and `std::false_type` are `std::integral_constant`.

Reply:
This is better. Thank you for suggestions